packages/amplication-server/src/core/workspace/workspace.resolver.ts

```diff
@@ -88,11 +88,6 @@ export class WorkspaceResolver {
       currentUser
     );
 
-    await this.workspaceService.migrateWorkspace(
```

Comment:
Are we confident that we finished with the old migration and we no longer need this lazy migration?